blog/content/post/2024-03-08-cross2024-ftpm-tee-ta.md

```diff
@@ -45,24 +45,34 @@ in a protected execution environment called a trusted execution environment
 For Arm Cortex-A, there exists the Arm TrustZone technology.
 When used on an embedded device it creates two distinct memory "worlds": a
 Normal World for the Operating System (referred to as Rich OS in documentation)
-and a Secure World for implementing the Trusted Execution Environment (TEE).
+and a Secure World, perfect for implementing the Trusted Execution
+Environment<sup>[[1]](#figure-1%3A-arm-trustzone-for-arm-cortex-a)</sup>. <!-- markdownlint-disable-line MD033 MD051 MD013 -->
```

Comment:
@tym2k1 I know we are not publishing it on the 3mdeb blog, but local-preview doesn't render the figures. At least for me

Reply:
I'm aware. I formatted the images with inline HTML for them to be centered on the pdf sent to CROSSCON:
```html
<div style="text-align: center;"> <img src="../../static/img/TEE_ARM_Cortex-a_exception_levels.svg" alt="Cortex-A TrustZone Exception Levels"> </div>
```
```markdown
![Cortex-A TrustZone Exception Levels](/img/TEE_ARM_Cortex-a_exception_levels.svg)
```

I guess that our webpage view doesn't support HTML rendering. I can change it back to markdown or leave it as it is but I need info what's the decision.